packages/@ourworldindata/components/src/LabeledSwitch/LabeledSwitch.scss

```diff
@@ -1,9 +1,6 @@
-$dark-text: #5b5b5b;
```

Comment:
I can revert these "convention changes" I made by swapping out the top level definitions for inline usage of the global variables if you'd prefer, Marcel.

Reply:
No, I've done more of these now :)

Overall, I find it nice sometimes that colors are given a more meaningful name with local SCSS variables inside a component. I don't mind that, but I do mind using hex codes when color names are available and make more sense.